terra/executor/celery/executor.py

```diff
@@ -251,126 +250,56 @@ def __init__(self, service_info):
 
     return volume_map
 
-  @staticmethod
-  def reconfigure_logger(sender, **kwargs):
-    # FIXME no idea how to reset this
-    # setup the logging when a task is reconfigured; e.g., changing logging
-    # level or hostname
-
-    if settings.terra.zone == 'runner' or settings.terra.zone == 'task':
-      print("SGR - reconfigure runner/task logging")
-
-      # when the celery task is done, its logger is automatically reconfigured;
-      # use that opportunity to close the stream
-      if hasattr(sender, '_socket_handler'):
-        sender._socket_handler.close()
-
   @staticmethod
   def configure_logger(sender, **kwargs):
-    sender._hostname = settings.logging.server.hostname
-    sender._port = settings.logging.server.port
-
-    if settings.terra.zone == 'controller':
-      print("SGR - setting up controller logging")
-
-      super(CeleryExecutor, CeleryExecutor).configure_logger(sender, **kwargs)
-
-      # setup the listener
-      sender.tcp_logging_server = LogRecordSocketReceiver(sender._hostname, sender._port)
-      print('SGR - About to start TCP server...')
-
-      lp = threading.Thread(target=sender.tcp_logging_server.serve_until_stopped)
-      lp.setDaemon(True)
-      # FIXME can't actually handle a log message until logging is done configuring
-      lp.start()
-      # TODO do we need to join
-      #lp.join()
-    elif settings.terra.zone == 'runner' or settings.terra.zone == 'task':
-      print("SGR - setting up runner/task logging")
-
-      sender._socket_handler = logging.handlers.SocketHandler(sender._hostname,
-          sender._port)
-
-      # TODO don't bother with a formatter, since a socket handler sends the event
-      # as an unformatted pickle
-
-      # FIXME this is hacky. it requires the executor know it is responsible for
-      # creating this variable on the logger
-      terra.logger._logs.main_log_handler = sender._socket_handler
+    if settings.terra.zone == 'task':
+      print(f"SGR - Not setting up CeleryExecutor:task logging")
+      sender.main_log_handler = NullHandler()
```

Comment:
I'd add a comment that this will be configured in `reconfigure_logger`. And note the problem: the celery workers are created when celery is started and they will be configured at that point. It is only when there is a work task to do that they get re-configured when the task starts and destroyed when the task is done. These workers can then get re-used for other tasks